actioncable/app/assets/javascripts/action_cable.js

```diff
@@ -1,153 +1,113 @@
 (function(global, factory) {
-  typeof exports === "object" && typeof module !== "undefined" ? factory(exports) : typeof define === "function" && define.amd ? define([ "exports" ], factory) : factory(global.ActionCable = {});
```

Comment:
Is there any way to introduce a deprecation message on this file telling people to use the new file?

Reply:
Good thinking. We probably could insert a console.log() statement in just the legacy output. It's just that such a statement would leak into the public app, if someone didn't notice in dev. Maybe that's ok?

Reply:
I think that would be ok.